src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -145,14 +146,16 @@ Okta, Azure, and OneLogin have New Relic apps for both the original user model a
 
 **Recommended**: Download the full list of existing original user model users before choosing to import users. This will be a useful resource and serve as a backup, if you need it. 
 
-After downloading your original user model users, you can upload all users or just some of them. This step will create user records on the New Relic One user model. In a later step, you’ll be able to transition these users' assets. 
+After downloading your original user model users, you can upload all users or just some of them. We recommend reviewing this list of users and removing any users you don't want migrated (for example, people who no longer work at your organization). 
 
-The new user record that's created has the same login credentials: there is no need to reset passwords. If a user has a pending email verification status (pending being verified), that will also be transitioned over. 
+When you reupload your list of users and complete this step, it will create user records on the New Relic One user model. In a later step, you’ll be able to transition these users' assets. 
+
+The new user records created for your users have the same login credentials: there's no need to reset passwords. If a user has a pending email verification status (pending being verified), that will also be transitioned over. 
 
 Important tips: 
```

Comment:
@zuluecho9, elsewhere in this doc, I see just "Tips." Maybe we should use one approach?